addons/dragging/XEH_postInit.sqf

```diff
@@ -56,3 +56,27 @@ if (isNil "ACE_maxWeightCarry") then {
 }, {
     false
 }, [-1, [false, false, false]]] call CBA_fnc_addKeybind; // UNBOUND
+
+["ACE3 Common", QGVAR(carry), (localize LSTRING(CarryKeybind)), {
+    if (!alive ACE_player) exitWith {false};
+    if !([ACE_player, objNull, ["isNotDragging", "isNotCarrying"]] call EFUNC(common,canInteractWith)) exitWith {false};
+
+    // If we are drag/carrying something right now then just drop it:
+    if (ACE_player getVariable [QGVAR(isDragging), false]) exitWith {
+        [ACE_player, ACE_player getVariable [QGVAR(draggedObject), objNull]] call FUNC(dropObject);
+        false
+    };
+    if (ACE_player getVariable [QGVAR(isCarrying), false]) exitWith {
+        [ACE_player, ACE_player getVariable [QGVAR(carriedObject), objNull]] call FUNC(dropObject_carry);
+        false
+    };
+
+    private _cursor = cursorObject;
+    if ((isNull _cursor) || {(_cursor distance ACE_player) > 2.6}) exitWith {false};
+    if (!([ACE_player, _cursor] call FUNC(canCarry))) exitWith {false};
+
+    [ACE_player, _cursor] call FUNC(startCarry);
+    false
+}, {
+    false
```

Comment:
Change to an empty code block if it just returns `false`; that way an key up handler isn't even added.

Reply:
updated